google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -60,8 +63,9 @@ public String getStreamName() {
 
   /** Stream has already been finalized. */
   public static final class StreamFinalizedException extends StorageException {
-    protected StreamFinalizedException(String name, String message, Throwable cause) {
-      super(message, cause, name, ImmutableMap.of());
+    protected StreamFinalizedException(
+        Status invalidArgument, Metadata metadata, String name, String message, Throwable cause) {
```

Comment:
Change invalidArgument to just status. Is message and cause still used?

Reply:
Updated and nope, they are cleaned out now.